packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -389,7 +388,6 @@ class TimeoutInput extends Component {
 // ===================================================================
 
 const DEFAULT_CRON_PATTERN = '0 0 * * *'
-const DEFAULT_TIMEZONE = moment.tz.guess()
```

Comment:
Why this change?

Reply:
When creating a new schedule, the timezone should default to the browser timezone.

When editing a schedule without a timezone, it should be interpreted as the server timezone (historical reason).